src/main/java/com/xebia/incubator/xebium/SeleniumDriverFixture.java

```diff
@@ -456,6 +524,81 @@ private boolean executeDoCommand(final String methodName, final String[] values)
 		}
 	}
 
+	private String encryptString(String password) { //encrypt exposed passwords or information stub
```

Comment:
Got it!
On Jun 28, 2015 5:50 PM, "Arnout Engelen" <notifications@github.com> wrote:

> In src/main/java/com/xebia/incubator/xebium/SeleniumDriverFixture.java
> <https://github.com/xebia/Xebium/pull/122#discussion_r33427425>:
>
> > @@ -456,6 +524,81 @@ private boolean executeDoCommand(final String methodName, final String[] values)
> >  		}
> >  	}
> >
> > +	private String encryptString(String password) { //encrypt exposed passwords or information stub
>
> I think the encryption/decryption methods should move to their own
> fixture, outside of Xebium.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xebia/Xebium/pull/122/files#r33427425>.
>